src/rxdeep/index.js

```diff
@@ -0,0 +1,10 @@
+import { State } from 'rxdeep';
+import { useObservable } from 'rxjs-hooks';
+
+import { createApp, initialState } from '../common';
+
+const state = new State(initialState.count);
+const useCount = () => useObservable(() => state) || initialState.count;
+const useIncrement = () => () => { state.value += 1; };
```

Comment:
We also need to take care of `state.dummy`. Please try using `reducer` somehow.

Reply:
[just did](https://github.com/loreanvictor/will-this-react-global-state-work-in-concurrent-mode/commit/6f8d5963f7d3b9ecf6f45f8f0ee5ec4bb8233299). shouldn't GitHub automatically pull new commits?